### ISSUE TITLE
Local Ingest: File Collector Cluster Name Not Provided

### ISSUE THREAD

**Issue by p4pentest** [DataDog/KubeHound#272]
Hi, I am struggling to get the local ingest run with the latest version (Also same issue with last couple of releases).

I have generated a local dump using an older kubehound release version (v1.3.2) and then tried to ingest it using kubehound v1.5.1 and other older releases.

I am getting the below error when I try to ingest the local dump:
```
FATA[22:59:20] ingest build data: raw data ingest: ingestor dependency health check: 1 error occurred:
        * file collector cluster name not provided
```

**Version 1.5.1 and 1.5.0:** 
![image](https://github.com/user-attachments/assets/eed18551-bac9-464e-a095-74172194047d)

**Version 1.4.0:**
![image](https://github.com/user-attachments/assets/74fc5847-091e-4e3d-bca3-c409848636c3)

I am using the below:
**Kubehound Release:** v1.5.1 and previous
**OS:** Linux kali 6.8.11-amd64 https://github.com/DataDog/KubeHound/pull/1 SMP PREEMPT_DYNAMIC Kali 6.8.11-1kali2 (2024-05-30) x86_64 GNU/Linux
**Also tried in:** Linux cp 5.15.0-101-generic #111~20.04.1-Ubuntu SMP Mon Mar 11 15:44:43 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
**Kubectl:** Client Version: v1.31.1, Kustomize Version: v5.4.2

**Comment by jt-dd** [comment]
Hello,

with version `v1.5.1` can you try with the following `--cluster kubernetes-admin@kubernetes` ?

**Comment by p4pentest** [comment]
Thanks for the prompt response! Interestingly it worked!

I thought `--cluster <clusername>` was applicable for previous versions and not the latest one since it didn't show up in when I tried the `--help` flag. Might be worth adding that info.

![image](https://github.com/user-attachments/assets/dce52aff-b333-4b52-b101-a23a9b4eb50f)

**Comment by jt-dd** [comment]
Yes the `--cluster` is deprecated. It is a temporal fix while we push the new release with the fix.

**Comment by jt-dd** [comment]
Can you confirm the issue has been fixed in v1.5.3 ?

**Comment by p4pentest** [comment]
Sorry for the late response, recently used the latest version and did a local dump, I didn't face any issue. So can confirm its fixed. Thank you!
